src/home/time.asm

```diff
@@ -71,16 +71,16 @@ SetupTimer::
 	ld b, -68 ; Value for Normal Speed
 	call CheckForCGB
 	jr c, .set_timer
-	ldh a, [rKEY1]
-	and $80
+	ldh a, [rSPD]
+	and SPD_DOUBLE
 	jr z, .set_timer
 	ld b, $100 - 2 * 68 ; Value for CGB Double Speed
```

Comment:
Kind of off-topic, but I think this might as well be:

```asm
ld b, -68 * 2 ; Value for CGB Double Speed
```

Reply:
I have to append LOW() so that rgbasm doesn't complain:
`warning: src/home.asm(56) -> src/home/time.asm(77): [-Wtruncation]
    Expression must be 8-bit; use LOW() to force 8-bit`